packages/component-library-react/src/Button.tsx

```diff
@@ -46,7 +48,8 @@ export const Button = forwardRef(
         type={type || 'button'}
         {...restProps}
       >
-        {children}
+        {icon}
+        {icon ? <span className="utrecht-button__label">{children}</span> : children}
```

Comment:
@bddjong Ik had eigenlijk ook (of alleen) een expliciete `label` property in gedachten, dat maakt het iets voorspelbaarder, en dan kunnen gebruikers er vanuit gaan dat die class name bestaat. Nu zou dat alleen zijn voor buttons met icons.